src/main.cpp

```diff
@@ -78,62 +12,14 @@ int main() {
 	rightCamera.configure(640, 480, 30);
 	rightCamera.loadCalibration("calibration_mats/cam_mats_right", "calibration_mats/dist_coefs_right");
 
-	// setup the stereo camera
-	StereoCamera camera = StereoCamera(leftCamera, rightCamera);
-
-	// intialize the depth map pipeline step
-	DepthMap dpMap = DepthMap();
-
-	// setup windows
-	cv::namedWindow("left");
-	cv::namedWindow("right");
-	cv::namedWindow("disparity");
+	DisparityPipeline pipeline = DisparityPipeline(leftCamera, rightCamera);
```

Comment:
This can be declared with
`DisparityPipeline pipeline(leftCamera, rightCamera);`